routing/get_fees_paid.js

```diff
@@ -90,6 +93,56 @@ module.exports = (args, cbk) => {
       // Get node icons
       getIcons: ['validate', ({}, cbk) => getIcons({fs: args.fs}, cbk)],
 
+      // Determine the In public key to use
+      getInKey: ['validate', (({}, cbk) => {
+        // Exit early if no in query is specified
+        if (!args.in) {
+          return cbk();
+        }
+
+        return asyncMap(args.lnds, (lnd, cbk) => {
+          return findKey({lnd, query: args.in}, cbk);
+        },
+        (err, res) => {
+          if (!!err) {
+            return cbk(err);
+          }
+
+          if (res.length > 1) {
```

Comment:
wouldn't you want to unique the public key here, and see if that is more than one? Maybe they both have the same peer, so same match